client/my-sites/stats/stats-module/style.scss

```diff
@@ -487,21 +496,21 @@ ul.module-header-actions {
 	// 2: Much wider to show horizontal scrolling better (since the window loads scrolled to the left)
 	&::before,
 	&::after {
+		background-image: linear-gradient(to left, $transparent 0%, $white 90%);
 		content: "";
 		display: block;
 		position: absolute;
-			top: 0;
 			bottom: 16px; // 1
 			left: 0;
-		z-index: z-index( 'root', '.module-content-table::after' );
+			top: 0;
 		width: 20px;
-		background-image: linear-gradient(to left, $transparent 0%, $white 90%);
+		z-index: z-index( 'root', '.module-content-table::after' );
 	}
 
 	&::after {
-		right: 0;
+		background-image: linear-gradient(to right, $transparent 0%, $white 90%);
```

Comment:
Could we use long-content-fade here instead? https://github.com/Automattic/wp-calypso/blob/master/assets/stylesheets/shared/mixins/_long-content-fade.scss